internal/utils/utils.go

```diff
@@ -589,7 +594,7 @@ func getBucketSearchNextPageParam(jsonParsed *gabs.Container) string {
 
 // IsCommit returns true if the string matches commit format
 func IsCommit(s string) bool {
-	re := regexp.MustCompile(`\b[0-9a-f]{5,40}\b`)
+	re := regexp.MustCompile(`^\b[0-9a-f]{5,40}\b`)
```

Comment:
Before this change, it captured the sha in whatever position of the string, so `8.0.0-abcdef-SNAPSHOT` was matched. Therefore this block: https://github.com/elastic/e2e-testing/blob/master/e2e/_suites/fleet/ingest_manager_test.go#L153-L157, was incorrectly evaluated to true.